.gitignore

```diff
@@ -27,9 +27,6 @@ config/default/vsphere_tmp
 # Ignore sed backup files
 *-e
 
-# Tooling
-.bin/
```

Comment:
Hi @andrewsykim,

Please leave this as is. It isn't hurting anything, and I doubt anything ever put there should be committed anyway.